src/components/Accounts/Actions/FeeFineActions.js

```diff
@@ -762,7 +765,7 @@ class Actions extends React.Component {
         data: refunds,
         comment: 'refunded',
         open: actions.refundModal || (actions.refundMany && !isWarning),
-        initialValues: { ...initialValues, amount: calculateSelectedAmount(this.props.accounts, true) }
+        initialValues: { ...initialValues, amount: calculateRefundSelectedAmount(feeFineActions) }
```

Comment:
`calculateSelectedAmount` can calculate refund amount and this function is used in different places in order to set correct state for refund action. Need to update it.

Reply:
@maximdidenko, I added changes in `calculateSelectedAmount`. 
Now for all `actions` except `refund` should be used `calculateSelectedAmount` func.
and for `refund action` - `calculateRefundSelectedAmount`.